pkg/k8s/factory_functions.go

```diff
@@ -154,3 +175,323 @@ func copyObjToV1Namespace(obj interface{}) meta_v1.Object {
 	}
 	return ns.DeepCopy()
 }
+
+func listV1NetworkPolicies(client interface{}) func() (versioned.Map, error) {
+	k8sClient, ok := client.(kubernetes.Interface)
+	if !ok {
+		log.Panicf("Invalid resource type %s: expecting 'kubernetes.Interface'", reflect.TypeOf(client))
+	}
+	return func() (versioned.Map, error) {
+		m := versioned.NewMap()
+		lo := meta_v1.ListOptions{Limit: 50}
```

Comment:
What is the rationale for setting 50 for the limit here? Add a comment specifying why.